cfgov/regulations3k/models/pages.py

```diff
@@ -98,8 +103,13 @@ def regulation_results_page(self, request):
             pre_tags=['<strong>'],
             post_tags=['</strong>']).models(SectionParagraph)
         for hit in sqs:
+            try:
+                snippet = Markup(" ".join(hit.highlighted))
+            except TypeError as e:
+                logger.info("Query string {} produced a TypeError: {}".format(
+                    search_query, e))
+                continue
```

Comment:
@higs4281 with this change, if you do a search for ", you get the warning about no results, but the sidebar still shows the result count. If you click on one of the regulations, you get the "Showing X matches" but there are no matches shown.

What about using `hit.text` here instead?

```py
if hit.highlighted:
    snippet = " ".join(hit.highlighted)
else:
    snippet = hit.text

...
'snippet': Markup(snippet),
...
```

This works even if you remove [this](https://github.com/cfpb/cfgov-refresh/pull/4762/files#diff-5ac6519566563e4cf77d313d88876facR78) single character check. Are there cases where searching for single characters might be useful, e.g. "Z"? ([working link with my proposed change](http://localhost:8000/policy-compliance/rulemaking/regulations/search-regulations/results/?q=Z&results=25&order=relevance))